build/charts/antrea/templates/agent/clusterrole.yaml

```diff
@@ -180,3 +180,17 @@ rules:
       - get
       - list
       - watch
+  {{- if eq .Values.trafficEncryptionMode "ipsec" }}
+  - apiGroups:
+    - certificates.k8s.io
+    resources:
+      - certificatesigningrequests
+    verbs:
+      - get
+      - watch
+      - list
+      - update
+      - patch
```

Comment:
does agent need update and patch permission?

Reply:
removed the update/path/delete permissions.